Week3/week3HomeWork/Exercise1/amazingNaji.js

```diff
@@ -0,0 +1,22 @@
+function giveCompliment(myName) {
+    const compliment = [
+        "GREAT",
+        "Awesome",
+        "Fantastic",
+        "So COOL",
+        "Excellent",
+        "the best",
+        "AMAZING",
+        "Nice",
+        "Wonderful",
+        "Superhero"
+    ];
+    let randomCopliment = Math.floor(Math.random() * 10);
```

Comment:
Ah yes, this works, but what if there are more than 10 items (or less than 10) in the array. Does this code still works?

Reply:
then it give error . the exercise only 10 complement